doc/source/cookbook/global_minimum.rst

```diff
@@ -0,0 +1,64 @@
+Global Minimum Search
+--------------------
+
+This module implements a scheme for finding/approaching the conformational global minimum of a |Molecule|. The script accomplishes this by systematically varying all dihedral angles, going through the following steps in the process:
+
+1.    A list of bonds is created, filtering out all bonds are either terminal, part of a ring or have a bond order larger than 1.
+
+2.    The geometry of the molecule is optimized with the first dihedral angle set to 120, 0 and -120 degree; the lowest energy conformer is returned.
+
+3.    This process is repeated in an incremental manner for all valid dihedral angles found in step 1., each step starting from the lowest energy conformer of the previous step.
+
+4.    After all dihedral angles have been exhausted, the final geometry is returned.
+
+
+Optimizations are possible at various levels of theory, RDKit UFF being a cheap default. Alternatively, the geometry can be optimized at an arbitrary level of theory with the help of the PLAMS |Jobrunner|. Besides the input molecule an additional two arguments, at minimum, are required: A type object of a class derived from |Jobs| and a dictionary of all keyword arguments that should be passed to aforementioned job (e.g. the job |Settings|).
+See below for an exampling using |ADFJob| (DFT level)::
+
+    s = Settings()
+    s.input.Geometry
+    s.input.basis.type = 'DZP'
+    s.input.XC.GGA = 'PBE'
+    s.input.NumericalQuality = 'Good'
+
+    mol_in = Molecule('my_mol.xyz')
+    job_kwarg = {'settings': s, 'name': 'my_first_DFTJob'}
+    mol_out = global_minimum(mol_in, job_type=ADFJob, **job_kwarg)
+
+
+Depending on the |Job| class, it may be necessary to manually bind the |get_energy()| and |get_main_molecule()| functions to the jobs matching |Results| class, these two functions being used for reading energies and geometries, respectively. For |ADFJob| this is unnecessary, but |AMSJob| for example does not have the |get_energy()| function assigned by default.
```

Comment:
Unfortunately `|get_energy()|` won't work as you expect. The `| ... |` syntax works only because of looong list of shortcuts/aliases defined at the end of `doc/source/conf.py`. If something is not there, you need to use the explicit Sphinx syntax that points to exact class/function/method: ``` :meth:`~scm.plams.interfaces.adfsuide.adf.ADFResults.get_energy` ```. Alternatively, you could just put it as regular monospace font, without any link: ``` ``get_energy()`` ```